Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -8,19 +8,21 @@ class Lesson3
     {
         public static void Demo()
         {
-            string firstName = PromptString("What is your first name?");
+             static void GatherInfo(string userNumber)
```

Comment:
Don't use nested functions, move this outside of demo function, into the class scope. This makes code less maintainable because we're forced to read about all the details (function calls and implementation are in one place).